[2023-09-15 06:03:49,827] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: covid_analytics_load.trusted_load 2023-09-15T06:00:25.812553+00:00 [queued]>
[2023-09-15 06:03:49,861] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: covid_analytics_load.trusted_load 2023-09-15T06:00:25.812553+00:00 [queued]>
[2023-09-15 06:03:49,862] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-09-15 06:03:49,863] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2023-09-15 06:03:49,863] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-09-15 06:03:49,902] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): trusted_load> on 2023-09-15T06:00:25.812553+00:00
[2023-09-15 06:03:49,912] {standard_task_runner.py:52} INFO - Started process 290 to run task
[2023-09-15 06:03:49,955] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'covid_analytics_load', 'trusted_load', '2023-09-15T06:00:25.812553+00:00', '--job-id', '2', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/covid_analytics_dag.py', '--cfg-path', '/tmp/tmpm0xi357h', '--error-file', '/tmp/tmptkndhczx']
[2023-09-15 06:03:50,038] {standard_task_runner.py:77} INFO - Job 2: Subtask trusted_load
[2023-09-15 06:03:51,136] {logging_mixin.py:104} INFO - Running <TaskInstance: covid_analytics_load.trusted_load 2023-09-15T06:00:25.812553+00:00 [running]> on host 1e9a66893905
[2023-09-15 06:03:51,394] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=solano.d.castro@gmail.com
AIRFLOW_CTX_DAG_OWNER=Airflow
AIRFLOW_CTX_DAG_ID=covid_analytics_load
AIRFLOW_CTX_TASK_ID=trusted_load
AIRFLOW_CTX_EXECUTION_DATE=2023-09-15T06:00:25.812553+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-09-15T06:00:25.812553+00:00
[2023-09-15 06:04:29,874] {python.py:118} INFO - Done. Returned value was: None
[2023-09-15 06:04:29,911] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=covid_analytics_load, task_id=trusted_load, execution_date=20230915T060025, start_date=20230915T060349, end_date=20230915T060429
[2023-09-15 06:04:30,047] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-09-15 06:04:30,089] {local_task_job.py:146} INFO - Task exited with return code 0
